### PR TITLE
Update the terms of use

### PR DESCRIPTION
**Pull request by paroxp**
What
----

There are content changes on Google Document that are now factored into
the platform. This content should hopefully be updated for people.

How to review
-------------

- Sanity check
- Validate content with the Doc ( https://www.diffchecker.com/ ) is probably enough...

### CODE REVIEW

File: config/accounts/documents/terms-of-use.md [hunk 1]
```diff
@@ -1,87 +1,92 @@
...
+The GOV.UK PaaS Memorandum of Understanding has been agreed by your department or organisation. Any revision of these Terms of Use will also be issued as version update to the signatory of the Memorandum of Understanding.
 
--   keep your data secure and comply with Data Protection Legislation/GDPR.
+## Terms of Use
 
-**You agree to:**
+These terms apply to our and your service's use of GOV.UK PaaS.
 
--   ensure that your service has passed the government security accreditation and the service standard assessment, where necessary
+Before you host a live service or private data on GOV.UK PaaS, your government organisation must have accepted and signed the GOV.UK PaaS Memorandum of Understanding).
```
Comment (lukemalchergds) [inline config/accounts/documents/terms-of-use.md]: remove trailing bracket

File: config/accounts/documents/terms-of-use.md [hunk 2]
```diff
@@ -1,87 +1,92 @@
...
-Terms of Use
-------------
+You will need to read and agree to these Terms of Use before using GOV.UK PaaS.
 
-These terms apply to your and your service's use of GOV.UK PaaS.
+## Changes we've made to the Terms of Use
 
-Before you host a live service or private data on GOV.UK PaaS, your government organisation must have accepted and signed the GOV.UK PaaS data sharing and financial agreement (Memorandum of Understanding).
+**13 January 2017**
+Document approved
```
Comment (LeePorte) [inline config/accounts/documents/terms-of-use.md]: I'm not seeing line breaks when this is deployed on my pipeline.

Comment (paroxp) [reply config/accounts/documents/terms-of-use.md]: Noticed that yesterday :) Thanks!

File: config/accounts/documents/terms-of-use.md (hunk 1 above)
Comment (paroxp) [reply config/accounts/documents/terms-of-use.md]: :done:

**Pull request merged**